.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ executors:
     resource_class: large
   rust_macos: &rust_macos_executor
     macos:
-      xcode: 11.4
+      xcode: 11.7
```

Comment:
Would there be compatibility concerns with updating directly to XCode 14 instead, so we don’t have to worry about this again for longer?

Reply:
I am honestly not sure! We could try going to at least 12 and see if it has any impact?